_posts/2015-10-28-refugeehackathon.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "#refugeehackathon"
+date: "2015-10-28 07:00:00"
+icon: flag
+categories: hackathon
+authors: ["Jan"]
+---
+
+Lots of refugees arrived in Germany.
+Task like organising accomodations, food and clothing or supporting them with their onwards journey raises many challenges.
```

Comment:
Task like organising accommodation, food and clothing and supporting them with their onwards journey raises many challenges.